content/tsc.schema.json

```diff
@@ -154,6 +154,24 @@
           "type": "boolean",
           "title": "Show stats",
           "default": false
+        },
+        "sharing_url": {
+          "type": "string",
+          "format": "url",
+          "title": "YouTube Sharing URL",
+          "description": "You can retrieve this by clicking Share on the YouTube video and copying the provided link."
+        },
+        "tweet_text": {
+          "type": "string",
+          "title": "Pre-defined tweet text"
+        },
+        "email_subject": {
+          "type": "string",
+          "title": "Pre-defined email subject"
+        },
+        "email_body": {
+          "type": "string",
+          "title": "Pre-defined email body text"
```

Comment:
this is actually the field description (this is a schema, not the actual JSON file) – but I do have some dummy text in the json as well, will obviously change that when needed :)